libs/search-widget/src/index.js

```diff
@@ -0,0 +1,4 @@
+export { default as NucliaSearchBar } from './widgets/search-video-widget/SearchBar.svelte';
+export { default as NucliaSearchResults } from './widgets/search-video-widget/SearchResults.svelte';
+export { default as NucliaViewerWidget } from './widgets/viewer-widget/ViewerWidget.svelte';
```

Comment:
might be a good time to rename our widget folder as well as it's not a search-video at all anymore.

Reply:
maybe later, I am tired refactoring

Reply:
actually it is needed as `search-widget` is the expected dist folder